pkg/util/cert/cert.go

```diff
@@ -138,7 +138,7 @@ func GenerateSelfSignedCertKey(host string, alternateIPs []net.IP, alternateDNS
 	template := x509.Certificate{
 		SerialNumber: big.NewInt(1),
 		Subject: pkix.Name{
-			CommonName: fmt.Sprintf("%s@%d", host, time.Now().Unix()),
+			CommonName: host,
```

Comment:
many clients fail TLS connections if they see two different certs with the same serial number from the same signer (and they have a local memory of previously seen certs from a given signer). we're hard-coding serialnumber:1 here, and host is likely to be the same between local cluster setups. that's why the timestamp was added